packages/http-client-java/generator/http-client-generator-standard-test/src/test/java/type/array/ModelValueClientTest.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package type.array;
+
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+class ModelValueClientTest {
+
+    ModelValueClient client = new ArrayClientBuilder().buildModelValueClient();
+
+    @Test
+    @Disabled("type.array.InnerModel is in unnamed module of loader 'app'")
```

Comment:
I think the error is `java.lang.ClassCastException: class java.util.LinkedHashMap cannot be cast to class type.array.InnerModel`?

Reply:
Fixed by the [commit#3412085](https://github.com/weidongxu-microsoft/typespec/pull/1/commits/3412085617cff69c74c6973062dc2ac1c6a26707)